test/statistics.jl

```diff
@@ -96,6 +100,9 @@ end
 @test_approx_eq var([1], 1; mean=[2], corrected=false) [1.0]
 
 @test var(1:8) == 6.
+@test varm(1:8,1) == var(1:8)
```

Comment:
pssst @kshyatt hit `y` when you link to lines of code

https://github.com/JuliaLang/julia/blob/a9b71c615e2568ab235e1f559bb07cf1afc9eaad/base/statistics.jl#L180